solver/Cargo.toml

```diff
@@ -17,6 +17,7 @@ async-trait = "0.1"
 contracts = { path = "../contracts" }
 ethcontract = { version = "0.11", default-features = false }
 futures = "0.3"
+transaction-retry = { git = "https://github.com/gnosis/gp-transaction-retry.git", rev = "b12c37e0916854dbf046d14e17cc4a7765666610" }
```

Comment:
Have we not settled on a version of this yet?

Reply:
I haven't pushed a tag for it yet so no. When we make the next change in that repo we can set a version.